care/users/api/viewsets/users.py

```diff
@@ -4,13 +4,16 @@
 from rest_framework.decorators import action
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
-
+from rest_framework import mixins
+from rest_framework.viewsets import GenericViewSet
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.viewsets'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5683989)